app/config.js

```diff
@@ -45,7 +48,7 @@ module.exports = {
   redisUrl: process.env.REDIS_URL || 'localhost',
 
   // The port redis is running
-  redisPort: process.env.REDIS_PORT || 6379,
+  redisPort: process.env.REDIS_PORT || 6380,
```

Comment:
Just a reminder when you get round to it

Reply:
lol every time 🤦